### ISSUE TITLE
How to override portletName for TwinkleModule

### ISSUE THREAD

**Issue by Xi-Plus** [wikimedia-gadgets/twinkle-core#19]
I have override `portletName` in this way:
https://github.com/Xi-Plus/twinkle-zhwiktionary/blob/7582f78403302babe9cee5da635947dfe825376e/src/batchdelete.ts#L5
The text in the menu is still "D-batch".
```typescript
export class BatchDelete extends BatchDeleteCore {
	portletName = '批刪';
```
doesn't work too.

**Comment by siddharthvp** [comment]
Oh, you're right - it can't be changed because the core class defines the default and calls `addMenu()` in its constructor, so the derived class doesn't get a chance to change it.

**Comment by siddharthvp** [comment]
Fixed in f5f485b4c by adding a beforeAddMenu() method to TwinkleModule class. Now you can do this by:
```ts
export class BatchDelete extends BatchDeleteCore {
	beforeAddMenu() {
		this.portletName = '...'; // or msg('...')
	}
```
